framework/tlvf/test/tlvf_test.cpp

```diff
@@ -754,7 +754,13 @@ int test_all()
 
 int main(int argc, char *argv[])
 {
-    mapf::Logger::Instance().LoggerInit("tlvf_test");
```

Comment:
Is it possible to add the below to `LoggerInit()` as an optional argument? It can be useful for other unit tests so it will save us from duplicating this code.

Reply:
I instead gave Logger a default config that logs to stdout, in a separate commit.